RDMA.md

```diff
@@ -0,0 +1,174 @@
+RDMA Support
```

Comment:
Any reason to store this file in the code repo?

I think we should move this to the docs repo, just like the contents of TLS.md has been ported to https://github.com/valkey-io/valkey-doc/blob/main/topics/encryption.md.

Reply:
> Any reason to store this file in the code repo?
> 
> I think we should move this to the docs repo, just like the contents of TLS.md has been ported to https://github.com/valkey-io/valkey-doc/blob/main/topics/encryption.md.

By the way, `valkey/TLS.md` will be removed? I guess the two documents should be at the same repo.

Reply:
> By the way, `valkey/TLS.md` will be removed? I guess the two documents should be at the same repo.

It's a good question. The TLS information is duplicated in two places and the information is already diverging. The README.md contains some information like this too, like building and installing, which is duplicated in the documentation repo. It's useful to have something like that in README, so I'm not sure.

I guess the docs are mainly for the users and the code repo files like README are mainly for developers. Ideally.

@madolson do you have an idea where about what belongs in each repo?